script/gitlabci/job_base.yml

```diff
@@ -145,3 +145,41 @@
   tags:
     - amd
     - rocm
+
+# -------- clang ---------
+
+.basecfg_clang:
+  image: ubuntu:${ALPAKA_CI_UBUNTU_VER}
+  variables:
+    ALPAKA_CI_UBUNTU_VER: "20.04"
+    CC: clang
+    CXX: clang++
+    ALPAKA_CI_SANITIZERS: ""
+    ALPAKA_CI_ANALYSIS: "OFF"
+    ALPAKA_CI_TBB_VERSION: 2021.4.0
+    ALPAKA_CI_RUN_TESTS: "ON"
+    alpaka_ACC_CPU_B_SEQ_T_SEQ_ENABLE: "ON"
+    alpaka_ACC_CPU_B_SEQ_T_THREADS_ENABLE: "ON"
+    alpaka_ACC_CPU_B_TBB_T_SEQ_ENABLE: "ON"
+    alpaka_ACC_CPU_B_OMP2_T_SEQ_ENABLE: "ON"
+    alpaka_ACC_CPU_B_SEQ_T_OMP2_ENABLE: "ON"
+    alpaka_ACC_ANY_BT_OMP5_ENABLE: "OFF"
+    alpaka_ACC_ANY_BT_OACC_ENABLE: "OFF"
+    alpaka_ACC_GPU_CUDA_ENABLE: "OFF"
+    alpaka_ACC_GPU_CUDA_ONLY_MODE: "OFF"
+    alpaka_ACC_GPU_HIP_ENABLE: "OFF"
+    alpaka_ACC_GPU_HIP_ONLY_MODE: "OFF"
+  extends: .base
+  tags:
+    - cpuonly
+
+.base_clang_legacy:
+  extends: .basecfg_clang
+  tags:
+    - intel
```

Comment:
What is the intention of the flags? At the moment, you force the job to use the CPUs of the GPU runners. If you want to use a x86 CPU, you should take the tag `x86_64`. Also you should use the tag `cpuonly`. This avoids, that the limited GPU runner resources are used for a CPU only job.

Reply:
According to [https://hifis.net/doc/software/gitlab/ci/#specific-runners] `x86_64` is the same as `intel`, leaving AMD CPUs out of our testing setup. Am I understanding this wrong?

Also, is there documentation for the `cpuonly` tag?

Reply:
This documentation is partial updated/incomplete. Go to the runner settings* in GitLab and expand the runners section. This is our big CPU machine:

![image](https://user-images.githubusercontent.com/10682583/207638407-1950b011-8649-4bee-a01f-759ff1ce17b5.png)

* you need to be logged in, that the link is working: https://gitlab.com/hzdr/crp/alpaka/-/settings/ci_cd